src/APIv2/Controller.php

```diff
@@ -896,7 +896,7 @@ public static function getItemsAutosuggest(ServerRequestInterface $request)
 		$query = $request->getQueryParams();
 		$search = Validation::validateHasString($query, 'q');
 
-		$min = 3;
+		$min = 1;
```

Comment:
Why?

Reply:
because the item autosuggest is used to confirm the existence of the items inputed when creating a donation, if the autosuggest doesn't propose the particular item the user is inputing it will not allow to be added, thus I need to allow that all queries for autosuggest to be accepted